components/places/sql/create_shared_schema.sql

```diff
@@ -197,6 +197,8 @@ CREATE TABLE IF NOT EXISTS moz_bookmarks_synced(
     siteURL TEXT
 );
 
+CREATE INDEX IF NOT EXISTS moz_bookmarks_synced_urls ON moz_bookmarks_synced(placeId);
```

Comment:
This index lets `tagsByPlaceId` use a covering index, and it'll come in handy in #2501, too.